src/main/java/seedu/address/logic/commands/MarkAttendanceCommand.java

```diff
@@ -0,0 +1,80 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.List;
+
+import seedu.address.model.Model;
+import seedu.address.model.person.Person;
+
+/**
+ * Marks the attendance of persons in address book with studentClass and indexNumber corresponding to the
+ * user's arguments.
+ */
+public class MarkAttendanceCommand extends Command {
+
+    public static final String COMMAND_WORD = "markAtt";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Marks attendance of all students in the "
+                                               + "specified class with index numbers matching the user input\n"
+                                               + "Parameters: d/DATE (DD/MM/YYYY) "
+                                               + "c/CLASS (only one class should be entered)"
+                                               + "ind/INDEX_NUMBERS (index numbers of students, separated by spaces)\n"
+                                               + "Example: " + COMMAND_WORD + " d/26/10/2022 c/1A ind/1 5 8 10\n"
+                                               + "Example: " + COMMAND_WORD + " d/05/08/2022 c/1B ind/4 7 9 10";
+
+    private static final String MESSAGE_MARK_ATTENDANCE_SUCCESS = "Attendance for students have been marked.";
+    private static final String MESSAGE_MARK_ATTENDANCE_NONE = "No student's attendance has been marked.";
```

Comment:
Use case for this? If no students are input (all students are PRESENT!) then attendance is still marked (as all "1")